posts/polyglot.md

```diff
@@ -0,0 +1,109 @@
+# Polyglot disciplines
+
+## Problems
+
+One of the biggest problems with switching between programming languages is the context switching. Remembering what operator is used to perform a particular language essential function e.g. `;` and `.` for statement termination or `.` or `->` for object member access.
```

Comment:
Jumping straight into the problems feels a bit odd.  Could do with some sort of context/intro